### ISSUE TITLE
Problem with deep object and a shallow value

### ISSUE THREAD

**Issue by frontsideair** [kiliman/remix-typedjson#4]
Sorry to be the bearer of the bad news, but I may have stumbled upon another bug. This is the payload:

```tsx
  return typedjson(
    { data: [{ greeting: "hello", today: new Date() }], counter: 1 },
  );
```

And the StackBlitz link: https://stackblitz.com/edit/github-gbcwza-6tcdt7?file=app%2Froutes%2Ftest.tsx

`counter` is `null` and its meta is `"undefined"`. Doesn't change when it's a string either.

**Comment by kiliman** [comment]
Thanks, I've fixed the issue, but broke something else. I should have this working soon.

**Comment by kiliman** [comment]
Fixed in v0.0.8
